### ISSUE TITLE
Accessibility issue when removing gradual tempo change without using mouse

### ISSUE THREAD

**Issue by Daniel-R15** [musescore/MuseScore#23522]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

NVDA screen reader is used.
Create a score in the new score window.
Adding an instrument to a new score.
Editing notes.
Adding a gradual tempo change.
Adding a dynamics.
Using the Alt + horizontal arrow key combination helps to find the dynamics, but does not help to find the gradual tempo change to delete it.

[ Originally reported by user vlad-mus on musescore.org forum "Support and bug reports" on 2024-07-02: https://musescore.org/en/node/364747#comment-1248399 ]

### Supporting files, videos and screenshots

Link to video demonstration:
Bug_report_№1_(2024-07-02_15-20-26).mp4
https://disk.yandex.ru/i/cjJ6Sj-5f90Xeg

### What is the latest version of MuseScore Studio where this issue is present?

MuseScore Studio, version 4.3.2

### Regression

I was unable to check

### Operating system

Windows 10 (64 bit)

### Additional context

For further confirmation of details, refer to the OP vlad-mus https://musescore.org/en/user/2370413 

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by bkunda** [comment]
@shoogle can you please also check this one (Narrator, and regression status)

**Comment by jmwmusic** [comment]
This seems to be a duplicate of 17599.
